### ISSUE TITLE
Issue using LLM completions with Langchain

### ISSUE THREAD

**Issue by minemo** [josStorer/RWKV-Runner#96]
As the title says, I'm trying to use the API with Langchain to implement it into some chains.
The ChatOpenAI wrapper works fine and generates expected outputs, but when I try to use the normal OpenAI class, 
the API endpoint does not accept the request made by Langchain. The error thrown is:
```json
{"detail":[{"loc":["body","prompt"],"msg":"str type expected","type":"type_error.str"}]}
``` 
I've experimented a bit with wireshark and maybe found the problem.
The prompt sent by the tool itself and expected by the API is like this
```json
{
"prompt":"...",
"stream":true,
"model":"rwkv",
"max_tokens":500,
"temperature":1.2,
"top_p":0.5,
"presence_penalty":0.4,
"frequency_penalty":0.4,
"stop":"\n\nBob"
}
```
In comparison to the request that Langchain sends:
```json
{
"prompt": ["Write me a song about sparkling water."], 
"model": "rwkv", 
"temperature": 0.7, 
"max_tokens": 256, 
"top_p": 1, 
"frequency_penalty": 0, 
"presence_penalty": 0, 
"n": 1, 
"logit_bias": {}
}
```
The issue seems to be that the RWKV API only supports singular strings as prompt input and throws an error if that doesn't happen.
However both Langchain's and OpenAI's specification allow arrays of strings as prompt parameter, and because Langchain internally uses arrays for single prompts too, it doesn't work.

**Comment by josStorer** [comment]
I will add support for string arrays after some days

**Comment by gsk147** [comment]
更新了嘛兄弟

**Comment by kenzone** [comment]
openai api 的 prompt 或者 input 支持 string 或 array，  mark 一下等更新

**Comment by josStorer** [comment]
@kenzone input 是embeddings API用的，embeddings API一直支持数组功能，目前仅有/completion API没有完全支持数组

**Comment by kenzone** [comment]
> 

![image](https://github.com/josStorer/RWKV-Runner/assets/573877/5956f663-03cc-42d6-915f-778e0635e4e4)

实测embeddings是不支持数组，最新版1.3.8

**Comment by josStorer** [comment]
fixed in v1.3.9: https://github.com/josStorer/RWKV-Runner/releases/tag/v1.3.9

@kenzone 感谢反馈，我的类型定义编写失误
